src/modules/Elsa.Alterations/Endpoints/Alterations/DryRun/Endpoint.cs

```diff
@@ -0,0 +1,30 @@
+using Elsa.Abstractions;
+using Elsa.Alterations.Core.Contracts;
+using Elsa.Alterations.Core.Models;
+using JetBrains.Annotations;
+
+namespace Elsa.Alterations.Endpoints.Alterations.DryRun;
+
+/// <summary>
+/// Executes an alteration plan.
+/// </summary>
+[PublicAPI]
+public class DryRun(IWorkflowInstanceFinder workflowInstanceFinder) : ElsaEndpoint<AlterationWorkflowInstanceFilter, Response>
+{
+    /// <inheritdoc />
+    public override void Configure()
+    {
+        Post("/alterations/dry-run");
+        ConfigurePermissions("run:alterations");
+    }
+
+    /// <inheritdoc />
+    public override async Task HandleAsync(AlterationWorkflowInstanceFilter filter, CancellationToken cancellationToken)
```

Comment:
Wouldn't it be easier to add dryrun as an optional parameter to the alteration endpoint? Now we need to ensure these 2 endpoints stay the same.